src/main/drivers/rangefinder_hcsr04.h

```diff
@@ -19,4 +19,6 @@
 
 #include "drivers/rangefinder.h"
 
+#define RANGEFINDER_HCSR04_TASK_PERIOD_MS 50
```

Comment:
Note comment says 70ms, at some point actual value was changed but comment was not. Product spec of HRCSR04 http://www.micropik.com/PDF/HCSR04.pdf says 60ms.
Product spec of SRF10 https://proto-pic.co.uk/content/datasheets/SRF10.pdf says 65ms.

70ms works for both and gives some leeway. I don't think there is really much advantage in getting values at a higher rate.